compress/compress_with_closure.sh

```diff
@@ -2,6 +2,30 @@
 
 DIR=`dirname $0`
 
+URL="http://dl.google.com/closure-compiler/compiler-latest.zip"
```

Comment:
Sorry about that.  Yes, it can be changed to https.  I will update it
accordingly.

On Thu, Oct 15, 2015 at 5:58 AM, Nils Kenneweg <notifications@github.com>
wrote:

> In compress/compress_with_closure.sh
> <https://github.com/bitwiseshiftleft/sjcl/pull/245#discussion_r42109299>:
>
> > @@ -2,6 +2,30 @@
> >
> >  DIR=`dirname $0`
> >
> > +URL="http://dl.google.com/closure-compiler/compiler-latest.zip"
>
> I am unhappy with the http in the url here! Is there an https version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitwiseshiftleft/sjcl/pull/245/files#r42109299>.
>